src/Common/tests/System.Net/Sockets/Fletcher32.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.Sockets.Tests
+{
+    // Utility type for calculating a running Fletcher-32 checksum.
+    internal struct Fletcher32
```

Comment:
Depending on the test, either:
- 256 256-byte datagrams are being sent, for a total of 64Kb (UDP)
- 123456 bytes of data are being sent (TCP)

I don't want to come up with a static 64 or 123Kb pattern. This checksum is very simple, and works with whatever content is sent with high confidence.

The other alternative I'd be willing to consider is deterministically generating the expected data on both the send and receive ends (e.g. using `Random` and a known seed). Note that this would present some mild challenges for UDP if packets are received out-of-order.